dashboard/src/main/java/eu/seaclouds/platform/dashboard/ConfigParameters.java

```diff
@@ -16,8 +16,46 @@
  */
 package eu.seaclouds.platform.dashboard;
 
-public interface ConfigParameters {
-    public static String MONITOR_ENDPOINT = "http://localhost:8081";    // CHANGE THIS
```

Comment:
It was both issues. Tomcat was not loading the servlets properly, probably because of some dependency issue, but not sure. However I've been testing it with Jetty without any problem.